odata-protocol/11.4 Data Modification.md

```diff
@@ -1040,11 +1040,20 @@ through the `ContentIDSupported` property of the
 [`Capabilities.DeepUpdateSupport`](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Capabilities.V1.md#DeepUpdateSupportType)
 term, both defined in [OData-VocCap](#ODataVocCap).
 
+For each entity being updated or removed, clients MAY specify an [ETag](#UseofETagsforAvoidingUpdateConflicts) value obtained from a previous request. 
+If an ETag is provided that does not match the ETag value of the entity being updated or removed, 
+or if an ETag is provided when adding or updating an entity that does not currently exist, 
+then services that support ETags MUST NOT apply the change and instead report a 
+`412 Precondition Failed` error according to the appropriate [`continue-on-error`](#ErrorHandlingwhenUpdatingaCollectionofEntities) behavior. 
```

Comment:
This paragraph seems out of place and should better move to the "Error Handling ..." section.